src/deepsparse/engine.py

```diff
@@ -521,6 +525,92 @@ def _properties_dict(self) -> Dict:
         }
 
 
+class Context(object):
+    """
+    Contexts can be used to run multiple instances of the MultiModelEngine with the same
+    scheduler. This is useful for running multiple models concurrently in a server-like
+    environment.
```

Comment:
Can you elaborate here on why it's useful? e.g. "This allows one scheduler to manage the resources of the system effectively, keeping engines that are running different models from fighting over system resources."